legend-depot-model/src/main/java/org/finos/legend/depot/domain/project/dependencies/VersionDependencyReport.java

```diff
@@ -0,0 +1,77 @@
+//  Copyright 2021 Goldman Sachs
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+package org.finos.legend.depot.domain.project.dependencies;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.commons.lang.builder.EqualsBuilder;
+import org.apache.commons.lang.builder.HashCodeBuilder;
+import org.finos.legend.depot.domain.project.ProjectVersion;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class VersionDependencyReport
+{
+    @JsonProperty
+    List<ProjectVersion> transitiveDependencies = new ArrayList<>();
+    @JsonProperty
+    boolean valid = true;
```

Comment:
default should be invalid?

Reply:
no because sometimes versions might have no dependency and that would mean their default needs to be true, otherwise we might go down the wrong rabbit hole